docs/engineering/technical-planning.md

```diff
@@ -0,0 +1,36 @@
+# Technical Planning Docs
+## Request for Comment
+In the course of planning development, there are often scenarios where a technology or implementation decision fork is presented and an engineer wants to solicit ideas or feedback. 
+
+Writing a full-blown Architectural Decision Record may not appropriate as that document requires significant overhead in proposing answers that still may be largely unknown. In most cases, we only want to invest the time in writing an ADR only when a decision has been made and needs to be codified. Until then, it's a Decision Proposal, not a Decision Record.
```

Comment:
>In most cases, we only want to invest the time in writing an ADR only when a decision has been made and needs to be codified.

seems to be a redundant "only", perhaps remove to read as:

In most cases, we only want to invest the time in writing an ADR when a decision has been made and needs to be codified.

Reply:
thanks